tools/AzureRM/AzureRM.psd1

```diff
@@ -71,6 +71,7 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '4.0.0';
                @{ModuleName = 'AzureRM.DataFactoryV2'; RequiredVersion = '0.3.0'; }, 
                @{ModuleName = 'AzureRM.DataLakeAnalytics'; RequiredVersion = '4.0.0'; }, 
                @{ModuleName = 'AzureRM.DataLakeStore'; RequiredVersion = '5.0.0'; }, 
+			   @{ModuleName = 'AzureRM.DataMigration'; RequiredVersion = '0.1.0'; },
```

Comment:
This won't be included here until the Mandatory *And* Advisory issues are resolved.  However, we can release independently to the gallery when the *Mandatory* comments are addressed.

Reply:
you need to revert this change, as specified